### ISSUE TITLE
[regression] rules with "gateway" option non-functional on assigned OpenVPN interfaces on 17.7.1 

### ISSUE THREAD

**Issue by fraenki** [opnsense/core#1801]
I've just noticed that after upgrading to 17.7.1 some parts of my setup stopped working. Turns out the following changes in `/tmp/rules.debug` caused this issue:

```
pass out log  route-to ( em0_vlanXX X.X.X ) from {em0_vlanXX} to {!(em0_vlanXX:network)} keep state allow-opts label "let out anything from firewall host itself" 
pass out log  route-to ( ovpnc1 255.255.252.0 ) from {ovpnc1} to {!(ovpnc1:network)} keep state allow-opts label "let out anything from firewall host itself" 
pass out log  route-to ( em0_vlanYY Y.Y.Y ) from {em0_vlanYY} to {!(em0_vlanYY:network)} keep state allow-opts label "let out anything from firewall host itself" 
pass out log  route-to ( em0_vlanZZ Z.Z.Z.Z ) from {em0_vlanZZ} to {!(em0_vlanZZ:network)} keep state allow-opts label "let out anything from firewall host itself" 
```

I'm pretty sure that the rule for "ovpnc1" does actually break my setup. Maybe https://github.com/opnsense/core/commit/e0cad8c3 should be fixed to exclude such interfaces?

**Comment by fraenki** [comment]
Hi guys, just a quick reminder: this is not yet in any release. Any chance to get it merged for 17.7.4? :)

**Comment by fichtner** [comment]
Will very likely land in 17.7.4, yes. Not yet backported though.
